src/reducers/vms.js

```diff
@@ -171,3 +192,6 @@ const vms = actionReducer(initialState, {
 })
 
 export default vms
+export {
```

Comment:
Do we really need this export, becauase I don't see any use of it?

Reply:
The initialState exports are for the unit test.  I needed to create a skeleton store for the selectors so the test would work.  Without the store the selectors throw an exception and the build fails.